app/abilities/sac_cas/person_ability.rb

```diff
@@ -11,20 +11,31 @@ module SacCas::PersonAbility
 
   prepended do
     on(Person) do
-      class_side(:create_households).if_sac_mitarbeiter
+      class_side(:create_households).if_sac_employee
       permission(:read_all_people).may(:read_all_people, :show).everybody
       permission(:layer_and_below_full)
         .may(:index_invoices, :create_membership_invoice)
-        .if_sac_mitarbeiter
+        .if_sac_employee
       permission(:any)
         .may(:set_sac_family_main_person)
         .if_person_is_adult_and_all_household_members_writable
+      permission(:any).may(:show_remarks).if_employee
+      permission(:any).may(:manage_remark_geschaeftsstelle).if_sac_employee
+      permission(:any).may(:manage_remarks_sektion).if_section_employee
     end
 
-    def if_sac_mitarbeiter
+    def if_employee
+      if_sac_employee || if_section_employee
+    end
+
+    def if_sac_employee
       SacCas::SAC_MITARBEITER_ROLES.any? { |r| role_type?(r) }
     end
 
+    def if_section_employee
```

Comment:
```suggestion
    def if_section_functionary
```

Die meisten Sektionen haben keine angestellte, sondern einfach Funktionäre (Ehrenamtlich)